lighthouse-core/test/report/html/renderer/util-test.js

```diff
@@ -12,23 +12,9 @@ const sampleResult = require('../../../results/sample_v2.json');
 const NBSP = '\xa0';
 
 /* eslint-env jest */
-/* eslint-disable no-console */
 /* global URL */
 
 describe('util helpers', () => {
-  let origConsoleWarn;
```

Comment:
needed for old `displayValue` handling (#5099) but no need after #6767 and missed for removal in #7628